style.css

```diff
@@ -9,12 +9,23 @@
 
 /* mobile-nav */
 
+.head {
+  width: 100vw;
+  margin: 44px auto;
+  height: 50px;
+  display: flex;
+  justify-content: center;
+}
+
 .head .mobile-nav {
-  margin-top: 44px;
+  background: white;
   display: flex;
-  padding: 10px 35px 0 24px;
+  z-index: 4;
+  padding: 20px 20px;
+  width: 90vw;
   align-items: center;
   justify-content: space-between;
+  position: fixed;
 }
```

Comment:
- [ ] Kindly give the `mobile-nav` `top: 0;` so it sticks at the top of the page, and not a little bit down at the top of the page.